gui/static.config.js

```diff
@@ -4,26 +4,32 @@ import JssProvider from 'react-jss/lib/JssProvider'
 import { MuiThemeProvider, createMuiTheme, createGenerateClassName } from '@material-ui/core/styles'
 import theme from './src/theme' // Custom Material UI theme
 
+const getVersion = () => {
```

Comment:
How do you feel about calculating this once and storing it in a variable? We could then return the variable from the function e.g.

```javascript
// ...
const buildInfo = extractBuildInfo()

// ...
getData: () => buildInfo
```

Reply:
Yea this is what I felt was wrong initially. Now I refactored the function for buildInfo in utils; take a look at the commit below.